extensions/objects/templates/fraction-editor.component.spec.ts

```diff
@@ -57,14 +57,6 @@ describe('FractionEditorComponent', () => {
 
     expect(component.currentFractionValueIsValid).toBeTrue();
     expect(component.errorMessageI18nKey).toBe('');
-    expect(component.eventBus.emit).toHaveBeenCalledWith(
```

Comment:
Why drop these?

Reply:
We don't use the event bus for validation anymore.